composer.json

```diff
@@ -28,6 +33,12 @@
         }
     },
     "require-dev": {
-        "sllh/php-cs-fixer-styleci-bridge": "^2.1"
+        "sllh/php-cs-fixer-styleci-bridge": "^2.1",
+        "symfony/dependency-injection": "^2.8 || ^3.0",
+        "symfony/http-kernel": "^2.8 || ^3.0"
+    },
+    "suggest": {
+        "symfony/http-kernel": "For Symfony integration as a bundle",
+        "symfony/dependency-injection": "For Symfony integration as a bundle"
```

Comment:
You could suggest this one instead: http://docs.php-http.org/en/latest/integrations/symfony-bundle.html

As we use httplug, this must be the required bundle for a Symfony integration.